README.adoc

```diff
@@ -35,6 +35,10 @@ Vagrant plugin installation method:
 $ vagrant plugin install vagrant-service-manager
 ------------------------------------------------
 
+NOTE: If you are encountering problems with the plugin installation,
+refer to the https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst[ADB installation instructions]. There you can find
+more information on how to install Vagrant on various systems.
+
```

Comment:
How about:To resolve any issues with the plugin installation and for more information on how to install Vagrant on various systems, refer to the ADB installation instructions.